src/index.scss

```diff
@@ -463,6 +463,12 @@ header {
   margin-bottom: 0px;
 }
 
+code {
+  word-wrap: break-word !important;
```

Comment:
I believe we can omit the **word-wrap** property. Could you please test it by removing this field?